Validator.UnitTest/ValidatorTest.cs

```diff
@@ -221,5 +222,29 @@ public void IsEqual(string input, bool expected)
             var actual = Validator.Equals(input, "Foo");
             Assert.Equal(expected, actual);
         }
+
+        [Theory]
+        [InlineData("Not a JSON string", false)]
+        [InlineData("{\"username\":\"Admin\",\"uuid\":\"d27aa483-a08d-4b7f-a2f1-b22dcd4af678\"}", true)]
```

Comment:
Maybe we should simplify the input test data? A guid is sort of long. I think the original author has the right idea. See [this](https://github.com/chriso/validator.js/blob/master/test/validators.js#L755) test.